plugins/github-issues/src/components/GitHubIssues/IssueCard/IssueCard.tsx

```diff
@@ -58,14 +60,22 @@ export const IssueCard = (props: IssueCardProps) => {
     commentsCount,
   } = props;
 
+  const config = useApi(configApiRef);
+
+  const hostname = readGithubIntegrationConfigs(
+    config.getOptionalConfigArray('integrations.github') ?? [],
+  )[0].host;
+
+  const githubHost = hostname || 'github.com';
```

Comment:
Hm. This data is [originally sourced](https://github.com/backstage/backstage/blob/e3f5f3a471cc0c36cebde8569836064791ddc6ba/plugins/github-issues/src/components/GitHubIssues/GitHubIssues.tsx#L50) out of the [useGetIssuesByRepoFromGitHub](https://github.com/backstage/backstage/blob/e3f5f3a471cc0c36cebde8569836064791ddc6ba/plugins/github-issues/src/hooks/useGetIssuesByRepoFromGitHub.ts#L21) hook, which uses the client to [execute this query](https://github.com/backstage/backstage/blob/e3f5f3a471cc0c36cebde8569836064791ddc6ba/plugins/github-issues/src/api/gitHubIssuesApi.ts#L229).

Seems to me that [the Repository type](https://docs.github.com/en/graphql/reference/objects#issue) has a `url` property, that you could be getting and then using that to tack an `/issues` onto that. Then this code will work no matter what, irrespectively of what the integrations say.

Reply:
Looks like we already have an issues url in there. I just stripped the issues /# from the end which should have the same result you described.